largest-series-product/largest_series_product_test.rb

```diff
@@ -7,56 +7,8 @@
 # rubocop:disable Lint/ParenthesesAsGroupedExpression
 #
 class Seriestest < Minitest::Test
-  def test_digits
-    assert_equal (0..9).to_a, Series.new('0123456789').digits
-  end
-
-  def test_same_digits_reversed
-    skip
-    assert_equal (0..9).to_a.reverse, Series.new('9876543210').digits
-  end
-
-  def test_fewer_digits
-    skip
-    assert_equal (4..8).to_a.reverse, Series.new('87654').digits
-  end
-
-  def test_some_other_digits
-    skip
-    assert_equal [9, 3, 6, 9, 2, 3, 4, 6, 8], Series.new('936923468').digits
-  end
-
-  def test_slices_of_zero
-    skip
-    assert_equal [], Series.new('').digits
-  end
-
-  def test_slices_of_2
-    skip
-    series = Series.new('01234')
-    expected = [[0, 1], [1, 2], [2, 3], [3, 4]]
-    assert_equal expected, series.slices(2)
-  end
-
-  def test_other_slices_of_2
-    skip
-    series = Series.new('98273463')
-    expected = [[9, 8], [8, 2], [2, 7], [7, 3], [3, 4], [4, 6], [6, 3]]
-    assert_equal expected, series.slices(2)
-  end
-
-  def test_slices_of_3
-    skip
-    series = Series.new('01234')
-    expected = [[0, 1, 2], [1, 2, 3], [2, 3, 4]]
-    assert_equal expected, series.slices(3)
-  end
-
-  def test_other_slices_of_3
-    skip
-    series = Series.new('982347')
-    expected = [[9, 8, 2], [8, 2, 3], [2, 3, 4], [3, 4, 7]]
-    assert_equal expected, series.slices(3)
+  def test_required_method_called_largest_product
+    assert_respond_to Series.allocate, :largest_product
```

Comment:
It is the one test that specifies the signature that Sieve will have an instance method called `largest_product`.  The signature of the test itself is `object, method_name, message`.
```
(from gem minitest-5.7.0)                                                      
=== Implementation from Minitest::Assertions                                   
------------------------------------------------------------------------------ 
  assert_respond_to(obj, meth, msg = nil)                                      
                                                                               
------------------------------------------------------------------------------ 
                                                                               
Fails unless obj responds to meth.                                             
```

The documentation via RDoc demonstrates it well.

```
==== (Uses superclass method RDoc::AnyMethod: MiniTest::Assertions#assert_respond_to) 
------------------------------------------------------------------------------        
                                                                                      
Tests if the given Object responds to method.                                         
                                                                                      
An optional failure message may be provided as the final argument.                    
                                                                                      
  assert_respond_to("hello", :reverse)  #Succeeds                                     
  assert_respond_to("hello", :does_not_exist)  #Fails                                 
```

I could have allocated the object separately, and used a variable, but it is succinct enough here, or I could have instantiated it, knowing the signature we want, but it would then fail with an argument error for initialize, and I wasn't sure we wanted to start driving the initialize method here.  Though of course we could.  If the important thing is the method itself, better to have it only address exactly what we want to address.

It is perhaps not the first test that should be ran, though.  Perhaps it should be the tests that would drive the creation of the `initialize` method.